### PR TITLE
LI-113/setup-schedule-link-app-repo

### PR DESCRIPTION
**Pull request by GeekEast**
- setup rollbar
- setup buildkite and github
- setup apollo, theme provider

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -27,4 +27,4 @@ dist-ssr
 coverage/
 .env
 build/
-dev-certs/
```
Comment (bennyli519) [inline .gitignore:30]: why we need to upload `dev-certs`

File: src/constants/phone.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import { CountryCode } from "libphonenumber-js"
```
Comment (bennyli519) [inline src/constants/phone.ts:1]: do we need this file?

File: .gitignore (hunk 1 above)
Comment (GeekEast) [reply .gitignore:30]: yeah I think it's not.

but I found in unit test. it is required for the vite.config.ts. to work.

I still don't think we need that in cicd pipeline. let me improve next PR to remove this and change vitest to jest.

File: src/constants/phone.ts (hunk 2 above)
Comment (GeekEast) [reply src/constants/phone.ts:1]: I include this one since it's used in constant file. copy from talenthub side.

Comment (bennyli519) [reply src/constants/phone.ts:1]: I see, let's leave it first

**Review by bennyli519 — APPROVED** [review]

File: src/constants/apolloVars.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { makeVar } from "@apollo/client"
+
+export enum SessionTimeoutNotificationMethod {
+  MODAL = "modal",
+  TOAST = "toast",
+}
+
+export const sessionHasTimeoutApolloVar = makeVar({
+  sessionHasTimeout: false,
```
Comment (yingzhi-lu) [inline src/constants/apolloVars.ts:9]: Do we need these? I remember we don't need session for candidate selection page

**Review by yingzhi-lu — APPROVED** [review]
LGTM! We can simplify the file during development

File: src/constants/apolloVars.ts (hunk 3 above)
Comment (GeekEast) [reply src/constants/apolloVars.ts:9]: no need. I will delete it in next PR shortly.

**Pull request merged**